RedisTopicListenerTest.startTimeBefore failed test

<!--  Thanks for submitting a bug report!  Before submitting:
1. Try searching the existing issues to see if your issue has already been reported
2. Try asking on our Discord group (https://discordapp.com/invite/FFb9YFX)
3. If you're reporting a security vulnerability, please email security@hedera.com instead of opening an issue
-->

**Detailed Description**
`RedisTopicListenerTest.startTimeBefore` failed in CircleCI.

```
[INFO] Running com.hedera.mirror.grpc.listener.RedisTopicListenerTest
2021-01-31T23:26:44.457-0600 INFO main c.h.m.g.l.RedisTopicListener Subscribing: TopicMessageFilter(endTime=null, limit=0, realmNum=0, startTime=1970-01-01T00:00:00Z, subscriberId=BjsldATB, topicNum=0) 
2021-01-31T23:26:44.463-0600 INFO main c.h.m.g.l.RedisTopicListener Creating shared subscription to topic.0.0.0 
2021-01-31T23:26:45.032-0600 INFO main c.h.m.g.l.RedisTopicListener Attempting to connect to Redis 
2021-01-31T23:26:45.033-0600 INFO main c.h.m.g.l.RedisTopicListener Connected to Redis 
2021-01-31T23:26:45.224-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:14.353079Z, realmNum=0, runningHashVersion=2, sequenceNumber=1, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:45.476-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:14.353079001Z, realmNum=0, runningHashVersion=2, sequenceNumber=2, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:45.484-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:14.353079002Z, realmNum=0, runningHashVersion=2, sequenceNumber=3, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:45.490-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:14.353079003Z, realmNum=0, runningHashVersion=2, sequenceNumber=4, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:45.496-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:14.353079004Z, realmNum=0, runningHashVersion=2, sequenceNumber=5, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:45.503-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:14.353079005Z, realmNum=0, runningHashVersion=2, sequenceNumber=6, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:45.513-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:14.353079006Z, realmNum=0, runningHashVersion=2, sequenceNumber=7, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:45.523-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:14.353079007Z, realmNum=0, runningHashVersion=2, sequenceNumber=8, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:45.531-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:14.353079008Z, realmNum=0, runningHashVersion=2, sequenceNumber=9, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:45.540-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:14.353079009Z, realmNum=0, runningHashVersion=2, sequenceNumber=10, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:45.702-0600 INFO boundedElastic-1 c.h.m.g.l.RedisTopicListener Unsubscribing from topic.0.0.0 
2021-01-31T23:26:45.824-0600 INFO main c.h.m.g.l.RedisTopicListener Subscribing: TopicMessageFilter(endTime=null, limit=0, realmNum=0, startTime=2021-02-01T05:27:15.749214Z, subscriberId=aJrrptOC, topicNum=0) 
2021-01-31T23:26:45.826-0600 INFO main c.h.m.g.l.RedisTopicListener Creating shared subscription to topic.0.0.0 
2021-01-31T23:26:45.886-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:15.749214Z, realmNum=0, runningHashVersion=2, sequenceNumber=1, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:45.894-0600 INFO boundedElastic-1 c.h.m.g.l.RedisTopicListener Unsubscribing from topic.0.0.0 
2021-01-31T23:26:46.047-0600 INFO main c.h.m.g.l.RedisTopicListener Subscribing: TopicMessageFilter(endTime=null, limit=0, realmNum=0, startTime=1970-01-01T00:00:00Z, subscriberId=ClHaKRI2, topicNum=1) 
2021-01-31T23:26:46.049-0600 INFO main c.h.m.g.l.RedisTopicListener Creating shared subscription to topic.0.0.1 
2021-01-31T23:26:46.056-0600 INFO main c.h.m.g.l.RedisTopicListener Subscribing: TopicMessageFilter(endTime=null, limit=0, realmNum=0, startTime=1970-01-01T00:00:00Z, subscriberId=ubgwZTFA, topicNum=2) 
2021-01-31T23:26:46.057-0600 INFO main c.h.m.g.l.RedisTopicListener Creating shared subscription to topic.0.0.2 
2021-01-31T23:26:46.117-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:15.966136001Z, realmNum=0, runningHashVersion=2, sequenceNumber=1, topicNum=1, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:46.126-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:15.966136002Z, realmNum=0, runningHashVersion=2, sequenceNumber=2, topicNum=1, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:46.136-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:15.966136003Z, realmNum=0, runningHashVersion=2, sequenceNumber=7, topicNum=2, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:46.145-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:15.966136004Z, realmNum=0, runningHashVersion=2, sequenceNumber=8, topicNum=2, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:46.153-0600 INFO boundedElastic-2 c.h.m.g.l.RedisTopicListener Unsubscribing from topic.0.0.2 
2021-01-31T23:26:46.155-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:15.966136005Z, realmNum=0, runningHashVersion=2, sequenceNumber=3, topicNum=1, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:46.171-0600 INFO boundedElastic-1 c.h.m.g.l.RedisTopicListener Unsubscribing from topic.0.0.1 
2021-01-31T23:26:46.351-0600 INFO main c.h.m.g.l.RedisTopicListener Subscribing: TopicMessageFilter(endTime=null, limit=0, realmNum=0, startTime=1970-01-01T00:00:00Z, subscriberId=HtuyLEEt, topicNum=0) 
2021-01-31T23:26:46.353-0600 INFO main c.h.m.g.l.RedisTopicListener Creating shared subscription to topic.0.0.0 
2021-01-31T23:26:46.413-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:16.247031Z, realmNum=0, runningHashVersion=2, sequenceNumber=1, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:46.422-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:16.247031001Z, realmNum=0, runningHashVersion=2, sequenceNumber=2, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:46.430-0600 INFO boundedElastic-1 c.h.m.g.l.RedisTopicListener Unsubscribing from topic.0.0.0 
2021-01-31T23:26:46.573-0600 INFO main c.h.m.g.l.RedisTopicListener Subscribing: TopicMessageFilter(endTime=null, limit=0, realmNum=0, startTime=1970-01-01T00:00:00Z, subscriberId=dISVQPKx, topicNum=1) 
2021-01-31T23:26:46.574-0600 INFO main c.h.m.g.l.RedisTopicListener Creating shared subscription to topic.0.0.1 
2021-01-31T23:26:46.634-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:16.492827001Z, realmNum=0, runningHashVersion=2, sequenceNumber=1, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:46.643-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:16.492827002Z, realmNum=0, runningHashVersion=2, sequenceNumber=2, topicNum=1, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:46.652-0600 INFO boundedElastic-1 c.h.m.g.l.RedisTopicListener Unsubscribing from topic.0.0.1 
2021-01-31T23:26:46.654-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:16.492827003Z, realmNum=0, runningHashVersion=2, sequenceNumber=3, topicNum=2, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:46.795-0600 INFO main c.h.m.g.l.RedisTopicListener Subscribing: TopicMessageFilter(endTime=null, limit=0, realmNum=1, startTime=1970-01-01T00:00:00Z, subscriberId=4d4n4aRg, topicNum=0) 
2021-01-31T23:26:46.796-0600 INFO main c.h.m.g.l.RedisTopicListener Creating shared subscription to topic.0.1.0 
2021-01-31T23:26:46.860-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:16.723067001Z, realmNum=0, runningHashVersion=2, sequenceNumber=1, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:46.869-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:16.723067002Z, realmNum=1, runningHashVersion=2, sequenceNumber=2, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:46.877-0600 INFO boundedElastic-1 c.h.m.g.l.RedisTopicListener Unsubscribing from topic.0.1.0 
2021-01-31T23:26:46.880-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:16.723067003Z, realmNum=2, runningHashVersion=2, sequenceNumber=3, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:47.026-0600 INFO main c.h.m.g.l.RedisTopicListener Subscribing: TopicMessageFilter(endTime=null, limit=0, realmNum=0, startTime=1970-01-01T00:00:00Z, subscriberId=Y2e0TmUo, topicNum=0) 
2021-01-31T23:26:47.027-0600 INFO main c.h.m.g.l.RedisTopicListener Creating shared subscription to topic.0.0.0 
2021-01-31T23:26:47.087-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:16.947959Z, realmNum=0, runningHashVersion=2, sequenceNumber=1, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:47.096-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:16.947959001Z, realmNum=0, runningHashVersion=2, sequenceNumber=2, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:47.104-0600 INFO boundedElastic-1 c.h.m.g.l.RedisTopicListener Unsubscribing from topic.0.0.0 
2021-01-31T23:26:47.249-0600 INFO main c.h.m.g.l.RedisTopicListener Subscribing: TopicMessageFilter(endTime=null, limit=0, realmNum=0, startTime=2021-02-01T05:27:17.170760Z, subscriberId=XulKEdrf, topicNum=0) 
2021-01-31T23:26:47.250-0600 INFO main c.h.m.g.l.RedisTopicListener Creating shared subscription to topic.0.0.0 
2021-01-31T23:26:47.360-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:17.170759999Z, realmNum=0, runningHashVersion=2, sequenceNumber=1, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:47.365-0600 INFO main c.h.m.g.l.RedisTopicListener Unsubscribing from topic.0.0.0 
2021-01-31T23:26:47.502-0600 INFO main c.h.m.g.l.RedisTopicListener Subscribing: TopicMessageFilter(endTime=null, limit=0, realmNum=0, startTime=1970-01-01T00:00:00Z, subscriberId=YWd5T0pl, topicNum=0) 
2021-01-31T23:26:47.504-0600 INFO main c.h.m.g.l.RedisTopicListener Creating shared subscription to topic.0.0.0 
2021-01-31T23:26:47.564-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:17.430159Z, realmNum=0, runningHashVersion=2, sequenceNumber=1, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:47.572-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:17.430159001Z, realmNum=0, runningHashVersion=2, sequenceNumber=2, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:47.580-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:17.430159002Z, realmNum=0, runningHashVersion=2, sequenceNumber=3, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:47.587-0600 INFO boundedElastic-1 c.h.m.g.l.RedisTopicListener Unsubscribing from topic.0.0.0 
2021-01-31T23:26:47.733-0600 INFO main c.h.m.g.l.RedisTopicListener Subscribing: TopicMessageFilter(endTime=null, limit=0, realmNum=0, startTime=1970-01-01T00:00:00Z, subscriberId=MalRZwBF, topicNum=0) 
2021-01-31T23:26:47.734-0600 INFO main c.h.m.g.l.RedisTopicListener Creating shared subscription to topic.0.0.0 
2021-01-31T23:26:48.233-0600 INFO parallel-2 c.h.m.g.l.RedisTopicListener Unsubscribing from topic.0.0.0 
2021-01-31T23:26:48.386-0600 INFO main c.h.m.g.l.RedisTopicListener Subscribing: TopicMessageFilter(endTime=null, limit=0, realmNum=0, startTime=1970-01-01T00:00:00Z, subscriberId=X7Rk9Kmc, topicNum=0) 
2021-01-31T23:26:48.387-0600 INFO main c.h.m.g.l.RedisTopicListener Creating shared subscription to topic.0.0.0 
2021-01-31T23:26:48.394-0600 INFO main c.h.m.g.l.RedisTopicListener Subscribing: TopicMessageFilter(endTime=null, limit=0, realmNum=0, startTime=1970-01-01T00:00:00Z, subscriberId=gUcLvTJr, topicNum=0) 
2021-01-31T23:26:48.412-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957Z, realmNum=0, runningHashVersion=2, sequenceNumber=1, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.419-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957001Z, realmNum=0, runningHashVersion=2, sequenceNumber=2, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.427-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957002Z, realmNum=0, runningHashVersion=2, sequenceNumber=3, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.435-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957003Z, realmNum=0, runningHashVersion=2, sequenceNumber=4, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.443-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957004Z, realmNum=0, runningHashVersion=2, sequenceNumber=5, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.451-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957005Z, realmNum=0, runningHashVersion=2, sequenceNumber=6, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.458-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957006Z, realmNum=0, runningHashVersion=2, sequenceNumber=7, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.465-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957007Z, realmNum=0, runningHashVersion=2, sequenceNumber=8, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.473-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957008Z, realmNum=0, runningHashVersion=2, sequenceNumber=9, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.481-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957009Z, realmNum=0, runningHashVersion=2, sequenceNumber=10, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.489-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957010Z, realmNum=0, runningHashVersion=2, sequenceNumber=11, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.496-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957011Z, realmNum=0, runningHashVersion=2, sequenceNumber=12, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.504-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957012Z, realmNum=0, runningHashVersion=2, sequenceNumber=13, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.512-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957013Z, realmNum=0, runningHashVersion=2, sequenceNumber=14, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.519-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957014Z, realmNum=0, runningHashVersion=2, sequenceNumber=15, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.527-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957015Z, realmNum=0, runningHashVersion=2, sequenceNumber=16, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.535-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957016Z, realmNum=0, runningHashVersion=2, sequenceNumber=17, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.542-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957017Z, realmNum=0, runningHashVersion=2, sequenceNumber=18, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.550-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957018Z, realmNum=0, runningHashVersion=2, sequenceNumber=19, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.557-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957019Z, realmNum=0, runningHashVersion=2, sequenceNumber=20, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.564-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957020Z, realmNum=0, runningHashVersion=2, sequenceNumber=21, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.571-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957021Z, realmNum=0, runningHashVersion=2, sequenceNumber=22, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.579-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957022Z, realmNum=0, runningHashVersion=2, sequenceNumber=23, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.586-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957023Z, realmNum=0, runningHashVersion=2, sequenceNumber=24, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.595-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957024Z, realmNum=0, runningHashVersion=2, sequenceNumber=25, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.603-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957025Z, realmNum=0, runningHashVersion=2, sequenceNumber=26, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.610-0600 TRACE main c.h.m.g.d.DomainBuilder Inserted: TopicMessage(consensusTimestampInstant=2021-02-01T05:27:18.301957026Z, realmNum=0, runningHashVersion=2, sequenceNumber=27, topicNum=0, chunkNum=null, chunkTotal=null, payerAccountEntity=null, validStartTimestamp=null, validStartInstant=null) 
2021-01-31T23:26:48.616-0600 INFO boundedElastic-1 c.h.m.g.l.RedisTopicListener Unsubscribing from topic.0.0.0 
[ERROR] Tests run: 11, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 4.354 s <<< FAILURE! - in com.hedera.mirror.grpc.listener.RedisTopicListenerTest
[ERROR] com.hedera.mirror.grpc.listener.RedisTopicListenerTest.startTimeBefore  Time elapsed: 1.357 s  <<< FAILURE!
java.lang.AssertionError: VerifySubscriber timed out on reactor.core.publisher.FluxMapFuseable$MapFuseableSubscriber@2587fd19

```

**Actual Behavior**

**Expected Behavior**
The test passes.

**Environment:**

**Additional Context**

--- a/hedera-mirror-grpc/src/test/java/com/hedera/mirror/grpc/listener/AbstractTopicListenerTest.java
+++ b/hedera-mirror-grpc/src/test/java/com/hedera/mirror/grpc/listener/AbstractTopicListenerTest.java
@@ -165,7 +165,7 @@ public abstract class AbstractTopicListenerTest extends GrpcIntegrationTest {
                 .then(() -> publish(domainBuilder.topicMessages(10, future)))
                 .expectNext(1L, 2L, 3L, 4L, 5L, 6L, 7L, 8L, 9L, 10L)
                 .thenCancel()
-                .verify(Duration.ofMillis(500));
+                .verify(Duration.ofMillis(1000));
     }
 
     @Test

